### PR TITLE
CCM-11324-added new markdown files for commitment orchestrator

### PR DESCRIPTION
**Pull request by charanya-j**
Preview build: https://650bfe29ed4b532a233efe92--harness-developer.netlify.app/docs/category/commitment-orchestrator

### CODE REVIEW

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+AWS offers two types of Reserved Instances: **Standard** and **Convertible**. The Convertible RIs provide more flexibility. You can set up Commitment Orchestrator for Convertible RIs. To learn more about the different types of Reserved Instances, go to [Types of Reserved Instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/reserved-instances-types.html).
+
+You can exchange your Convertible Reserved instance for another Convertible Reserved Instance with a different configuration, including instance family, operating system, and tenancy. There are no limits to how many times you perform an exchange, as long as the new Convertible Reserved Instance is of an equal or higher value than the Convertible Reserved Instances that you are exchanging. For more information, go to [Exchange Convertible Reserved Instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ri-convertible-exchange.html).
+
+For more information about Reserved Instances, go to [Reserved Instances. \
```
Comment (sandyydk) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md:52]: nit: Additional backslash here. This is visible in the docs.

**Review by sandyydk — CHANGES_REQUESTED** [review]
Looks good to me. Added a small comment to remove a typo. 

@navaneeth Do we want to retain the image used in docs for Setup Screen where projected savings look a bit less or is it fine as it's only a sample screen?

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md (hunk 1 above)
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md:52]: Noted.

**Review by jyotiarora-harness — APPROVED** [review]

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/use-commitment-orch.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    You can use the **Instance family** filter to view Instance types of a particular Instance family and the **Region** filter to view all Instance types available in that region. 
+
+    <docimage path={require('./static/setup-2.png')} width="50%" height="50%" title="Click to view full size image" />
+
+   4. Select **Continue**.
+   5. Carefully review the automated actions displayed on the **Review** page. You can also view the **Current Compute Spend**, **Target Compute Spend**, and **Projected Potential Savings** based on the setup. 
+
+   The following automated actions will be taken to optimize utilization:
+
+   - The Commitment Orchestrator continuously looks for opportunities to acquire or convert Reserved Instances (RIs)
```
Comment (Additi) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/use-commitment-orch.md:67]: Can we also add we do daily staggered purchases and not in bulk.

**Review by Additi — CHANGES_REQUESTED** [review]
PTAL

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/_category_.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "label":"Commitment Orchestrator",
+    "position": 3,
+    "collapsible":"true",
+    "collapsed":"true",
+    "className":"red",
+    "link":{
+       "type":"generated-index",
+       "title":"Optimize cloud costs by using Commitment Orchestrator"
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/_category_.json:9]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+sidebar_position: 2
+---  
+
+:::note
+Currently, this feature is behind the feature flag **CCM_COMMORCH**. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+:::
+
+On the summary page, you have the ability to perform several actions, including:
+
+* Viewing the total Compute spend, Compute coverage, Savings, and the Commitment utilization data.
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md:13]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+* Viewing data for the selected period grouped by the Commitment type, Instance Family, or Region in a bar graph. You can view detailed information for each day by hovering over the graph.
+* You can use the **Instance family** filter to view Instance types of a particular Instance family and the **Region** filter to view all Instance types available in that region. 
+
+
+You can gather the following details from this page:
+
+* **Compute spend**: The total amount spent on all compute instances that include Savings Plans, RIs, and On-Demand instances.
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md:20]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+* Viewing the total Compute spend, Compute coverage, Savings, and the Commitment utilization data.
+* Viewing data for the selected period grouped by the Commitment type, Instance Family, or Region in a bar graph. You can view detailed information for each day by hovering over the graph.
+* You can use the **Instance family** filter to view Instance types of a particular Instance family and the **Region** filter to view all Instance types available in that region. 
+
+
+You can gather the following details from this page:
+
+* **Compute spend**: The total amount spent on all compute instances that include Savings Plans, RIs, and On-Demand instances. 
+* **Compute Coverage**: This visualization illustrates the percentage of coverage attributed to specific commitment types. Within the bar graph, the X-axis denotes dates, while the Y-axis represents the corresponding costs. Beneath the graph, you can find a breakdown of costs for each commitment type, instance family, and region.
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md:21]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+* **Savings**: The total cost savings achieved by setting up the Commitment Orchestrator. Within the bar graph, the X-axis denotes dates, while the Y-axis represents the corresponding saved costs. Beneath the graph, you can find a breakdown of costs for each commitment type, instance family, and region.
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md:22]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+* **Commitment Utilization**: This visualization illustrates the percentage utilization of RIs and Savings Plans. Within the bar graph, the X-axis denotes dates, while the Y-axis represents the corresponding percentage utilization of the commitment type. Below the graph, you can find a detailed breakdown of each commitment type with its respective compute spend, cost utilization, and percentage utilization data.
+
+  <docimage path={require('./static/commitment-overview-1.gif')} width="50%" height="50%" title="Click to view full size image" />
+
+
+After setting up the Commitment Orchestrator, some automated actions are taken in order to optimize utilization of the RIs and Savings Plans. The details of these events are recorded and displayed in the **Log History**. Within the bar graph, the X-axis denotes dates, while the Y-axis represents the number of events. Beneath the graph, you can find the following event details:
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch-summary.md:28]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
+---
+title: Overview 
+description: An overview of Harness Commitment Orchestrator.
+sidebar_position: 1
+---  
+ 
+
+# Overview of Commitment Orchestrator
+
+The Harness Cloud Cost Management (CCM) platform provides powerful cost-saving mechanisms through its AutoStopping and Cluster Orchestrator features, which can automatically shut down non-production resources that are idle and run workloads on fully orchestrated spot instances. However, these mechanisms are not suitable for all workloads. Workloads that are not fault-tolerant, have unclear dependencies, lack high availability, or are stateful may not be compatible with these cost-saving mechanisms.
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md:10]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/use-commitment-orch.md (hunk 2 above)
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/use-commitment-orch.md:67]: Done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+In such cases, you can still optimize your resource utilization and save costs by making the most of your existing Reserved Instances (RI) before purchasing new ones. By carefully managing your RIs and ensuring that they're being utilized effectively, you can reduce your overall spending and make the most of your available resources.
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/commitment-orch.md:12]: Added this here: # How does Commitment Orchestrator optimize utilization?

**Comment by bot2-harness** [comment]
This PR has been automatically closed due to inactivity.

**Pull request closed without merging**